extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -131,6 +135,13 @@ define('ext.wikia.adEngine.adContext', [
 			!noExternals
 		);
 
+		// Floating medrec
+		context.opts.floatingMedrec = !!(
+			context.opts.showAds && context.opts.adsInContent &&
+			(isArticle() || context.targeting.pageType === 'search') &&
```

Comment:
Good point about moving this to isArticle function. What do you think about unifying the approach and creating one function to cover all page types?
for example:
isPageType('article'), isPageType('search') etc.